src/views/ArticleBody.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Tags from './ArticleTags';
 import clapImage from '../images/clap.svg';
+import SocailShare from '../components/SocialShare';
```

Comment:
#### Nitpick
Incorrect and inconsistency in spelling can introduce an unwanted bug to your code base. Kindly rename this to properly represent what it is doing.

Reply:
Yh thanks for that. I have changed it. https://github.com/andela/apollo-ah-frontend/blob/feature%2F164198153%2Fshare-article/src%2Fviews%2FArticleBody.jsx/#L5